avm-transpiler/src/transpile.rs

```diff
@@ -38,6 +38,9 @@ pub fn brillig_to_avm(brillig: &Brillig) -> Vec<u8> {
                 // TODO(4268): set in_tag to `field`
                 avm_instrs.push(AvmInstruction {
                     opcode: avm_opcode,
+                    indirect: Some(0),
+                    // TEMPORARY - instruction set currently expects this
+                    dst_tag: Some(AvmTypeTag::UINT32),
```

Comment:
Start this comment with `TODO(4268)` as it should be handled properly by https://github.com/AztecProtocol/aztec-packages/issues/4268